Expensify/App
# Save VBA flow information as users type

Users should have their VBA flow information saved as they type, so if they accidentally closed the pane or refresh the page, the data should show up again.

cc @marcaaron 

### Fixed Issues
$ https://github.com/Expensify/App/issues/5035

### Tests
1. Login to NewDot with an account that has a Workspace, but no bank account setup.
2. Navigate to `Settings > Select your Workspace > Get Started`
3. Go through the flow typing in your information. You can use [this SO](https://stackoverflow.com/c/expensify/questions/342) to get to the later steps.
4. Close the right pane or refresh the page periodically (before clicking on `Save & Continue` in each step).
5. Verify that information is not lost and all data is still there when you reopen the flow.

Testing potential Onyx race condition
1. When you get to the `Beneficial owners` step, check `Somebody else owns more than 25% of <companyName>`. Add 3 additional beneficial owners.
2. Open the browser console, select the `Application` tab and search for the `reimbursementAccountDraft` key under `localStorage`.
3. Back in the app, click `Remove this beneficial owner` and verify that the `beneficialOwners` array display the 2 remaining `beneficialOwners` and not `null`. Repeat the step until you are left with just one owner left.



### QA Steps
Steps above.

### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/22219519/132266210-6709eb82-d5b5-45bd-9c18-fae93fd20fba.mov

#### Mobile Web

https://user-images.githubusercontent.com/22219519/132266447-217b79c2-4df2-4a34-8457-db46b46f6770.mov

#### Desktop

https://user-images.githubusercontent.com/22219519/132266212-464796c8-1f14-4c54-a464-93aa7881362f.mov

#### iOS

https://user-images.githubusercontent.com/22219519/132266738-3baeb955-8812-41cb-9395-2c958fc88228.mov

#### Android

https://user-images.githubusercontent.com/22219519/132266215-95d55426-e645-4fef-8c9d-1af1dda8b92a.mov
